game-core/src/main/java/games/strategy/triplea/oddsCalculator/ta/BattleResults.java

```diff
@@ -9,13 +9,17 @@
 import games.strategy.triplea.delegate.IBattle.WhoWon;
 import games.strategy.triplea.delegate.Matches;
 import games.strategy.util.CollectionUtils;
+import lombok.Getter;
 
 public class BattleResults extends GameDataComponent {
   private static final long serialVersionUID = 1381361441940258702L;
 
-  private final int m_battleRoundsFought;
-  final List<Unit> m_remainingAttackingUnits;
```

Comment:
maybe worth pointing out, the annotations made variable renaming particularly easy. Generally useful to have fewer dependencies on any given name.